### ISSUE TITLE
Exchange Support

### ISSUE THREAD

**Issue by Thomas101** [wavebox/waveboxapp#268]
This issue tracks the upcoming exchange support and collapses the following issues....

1.) Redirect to company ADFS login screen #230 #13
2.) Use of Exchange Online (Plan 1) #259 
![](https://user-images.githubusercontent.com/29484676/27230049-9afb9de8-52b6-11e7-9493-99e6d6a18775.png)

**Comment by rmtilson** [comment]
Just want to touch base on #195 Basic Auth works for accessing Office 365 with ADFS.

**Comment by Thomas101** [comment]
@rmtilson does this mean you were able to login?

**Comment by rmtilson** [comment]
@Thomas101 Yes, I am able to login to Office 365. I get a pop up box now for my creds instead of electron or chromium trying to pass through creds and getting a IIS forbidden message.

However I did notice on the prompt that the server name is not displayed before ":443".

**Comment by NeckBeardPrince** [comment]
Looking forward to this getting resolved. Company just moved from Google to 365 and I need my Wavebox 💉

**Comment by Thomas101** [comment]
Just an update on this - from our testing we've found that you can come up against this error if the sysadmin has disabled adding 3rd party apps. We've put some steps together that will allow the sysadmin to grant permission for users to use Wavebox. There's some more info on the steps [in this KB article](https://wavebox.io/kb/office-365-admin-permission) if that helps
